stestr/cli.py

```diff
@@ -48,6 +48,11 @@ def build_option_parser(self, description, version, argparse_kwargs=None):
         return parser
 
     def _set_common_opts(self, parser):
+        parser.add_argument('--user-config', dest='user_config', default=None,
+                            help='An optional path to a default user config '
+                                 'file if one is not specified ~/.stestr.yml '
+                                 'and ~/.config/stestr.yml will be tried in '
```

Comment:
Oops, this should be ~/.stestr.conf and ~/.config/.stestr.conf (instead of .yml) that's what is implemented in code. I'll respin this quickly right now.